ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -480,6 +481,36 @@ Rectangle {
         return found[1]
     }
 
+    function getTextWithPublicKeys() {
+        let result = messageInputField.getText(0, messageInputField.length)
+
+        if (mentionsPos.length > 0) {
+            let lastIndex = messageInputField.length + 1
+            while (lastIndex > 0) {
```

Comment:
@faust4exe because of the previous line we always enter this while loop even for empty `messageInputField.text`

Reply:
We cannot enter if here - because when text is empty - the mentionsPos is also empty :) But ok it should work with `lastIndex = length` anyway